redaxo/src/addons/mediapool/lib/media_category_select.php

```diff
@@ -27,7 +30,7 @@ public function __construct($check_perms = true)
     /**
      * Kategorie-Id oder ein Array von Kategorie-Ids als Wurzelelemente der Select-Box.
      *
-     * @param mixed $rootId Kategorie-Id oder Array von Kategorie-Ids zur Identifikation der Wurzelelemente
+     * @param int|int[] $rootId Kategorie-Id oder Array von Kategorie-Ids zur Identifikation der Wurzelelemente
```

Comment:
```suggestion
     * @param int|int[]|null $rootId Kategorie-Id oder Array von Kategorie-Ids zur Identifikation der Wurzelelemente
```

Denke, hier sollte auch `null` erlaubt sein, um es zurücksetzen zu können.

Reply:
hatte ich auch überlegt.. war mir aber unsicher